ros2action/test/test_cli.py

```diff
@@ -251,8 +252,6 @@ def test_send_fibonacci_goal_with_feedback(self):
             assert action_command.wait_for_shutdown(timeout=10)
         assert action_command.exit_code == launch_testing.asserts.EXIT_OK
         assert launch_testing.tools.expect_output(
-            expected_lines=get_fibonacci_send_goal_output(
-                order=5, with_feedback=True
-            ),
-            text=action_command.output, strict=True
+            expected_lines=get_fibonacci_send_goal_output(order=5, with_feedback=1),
+            text=action_command.output, strict=False
```

Comment:
Why not `strict` anymore?

Reply:
TL; DR because we cannot guarantee all action feedback will be received.

This bit is covered by https://github.com/ros2/ros2cli/pull/494/commits/b82bcd697c9bc5359694a558dcbd4475051d6e41. Test was flaky because it expected all published goal feedback to be printed. But the `ros2 action send_goal` exits once the goal result arrives, but actions' design does not (it cannot) guarantee all feedback will have arrived by the time the result response arrives. So I had to change the test to only expect the first piece of expected feedback and then look for the result -- which still does not provide any guarantees but it's much less likely to fail.

Having said that, we could try to match as many feedback messages as possible. But `expect_output()` can't do that now.